check_pgactivity

```diff
@@ -1392,6 +1392,8 @@ sub nagios_strict_output ($$;$$$) {
 
     set_strict_units ( \@perfdata );
 
+    map { $$_[1] = 'U' if $$_[1] eq 'NaN' } @perfdata;
```

Comment:
How how instead changing `set_strict_units()` to say `set_strict_perfdata` that would be responsible of taking care of both units and NaN?

Reply:
You are right obviously. Fixed